### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD002.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#6] [General]
# https://blog.3place.cn/3place/fsd/FSD002.html

https://blog.3place.cn/3place/fsd/FSD002.html

**Comment by picklecai** [comment]
柱子撑好，戏台已经搭成，
人人都在等候盛会开始。

相当于：

啤酒饮料矿泉水，
花生瓜子八宝粥。
小板凳已经搬好了。

**Comment by picklecai** [comment]
大批观众
像潮水一般拥向我们的戏馆，
仿佛发出强烈的反复的阵痛，
争相挤进这个狭窄的善门

既然以潮水涌入的量争相挤进窄门，想来是痛的。门虽不痛，“我”替它痛了。
见到大量金子涌入的眼睛，居然见得到痛，意想不到。

**Comment by picklecai** [comment]
爱与友谊以神的手法
创造、培育我们心灵的造化。

→ 诗从情中来。没有情，就是无病呻吟，“为赋新词强说愁”。

**Comment by picklecai** [comment]
从我们内心深处涌出的诗潮，
在我们口边羞怯地哼出的诗篇，
有时很糟，有时或许很好。

“文章本天成，妙手偶得之”。以前以为是纯粹说好，其实是说大量的“表达自我”中，偶尔得了妙手。

**Comment by picklecai** [comment]
请给幻想添上一切合唱，
理性、理智、感情以及热情，
可是注意！不要把丑角遗忘。

谁会遗忘丑角？反正观众不会。丑角想不让观众注意都难。
所以，他说的是诗人、剧作者，不要忘了丑角，不要忘了：理性、理智、感情以及热情之外，还有幽默搞笑，还有自嘲讽刺。

剧作者为什么会容易遗忘丑角？因为他不是坐在台下看戏的，他只是想表达自己。要心里有观众，才不会忘了丑角。

**Comment by picklecai** [comment]
干吗你们这些蠢材
为这些家伙去麻烦缪斯？

这是文学创作的k策略和r策略。
不过文学的r策略，不指望能诞生出伟大来吧？

**Comment by picklecai** [comment]
他用什么感动人心？
他用什么将四大驱遣？
那不就是从胸中涌出而把世界
摄回到他心中的那种“和谐”？
……
都是靠诗人启示的人类的威力。

这是负熵。
生命是负熵，真正的创作也是负熵。
负熵总是艰难的，然而成果会让我们展颜一笑。

**Comment by picklecai** [comment]
谁都是过来人，却很少有人领悟，
一到你手里，就变得非常有趣。

所以苏轼、李白……不可复制

**Comment by picklecai** [comment]
丑角深谙什么能刺激起青春的荷尔蒙。
昨天的关键词是魅惑，今天的关键词就是青春吧？

**Comment by picklecai** [comment]
通过这狭隘的木棚，
请去跨越宇宙的全境，
以一种从容不迫的速度，
遍游天上人间和地狱。

灵台方寸山，斜月三星洞。
